.travis.yml

```diff
@@ -15,7 +15,7 @@ matrix:
     - node_js: '4'
       env: NO_WEB=
 script:
-  - test $NO_WEB && (npm run coverage && npm run publish-coverage) || test $SAUCE_USERNAME && npm run zuul || echo 'not running on saucelabs'
+  - test $NO_WEB && (npm run coverage && npm run publish-coverage) && ( test $SAUCE_USERNAME && npm run zuul || echo 'not running on saucelabs' )
```

Comment:
That was an old-school if. I think what was missing was a parentheses after the first `&&`.